user-guide/Advanced_Modules/Dashboards_app/Creating_and_configuring_dashboards/Sharing_a_dashboard.md

```diff
@@ -26,11 +26,13 @@ uid: Sharing_a_dashboard
 
 1. If it is the first time you are sharing the dashboard, you may be asked to confirm that you want to link your account to the cloud. Select *I want to link the above users* and click *Link accounts*.
 
-1. In the pop-up window, fill in the email address of the person you want to share the dashboard with.
+1. In the pop-up window, *With* field, fill in the email address of the person you want to share the dashboard with.
```

Comment:
I would reword this as:
"In the pop-up window, fill in the email address of the person(s) you want to share the dashboard with in the *With* field.